specification/eventgrid/data-plane/Microsoft.HealthcareApis/stable/2018-01-01/examples/fhir_resource_created.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "e4c7f556-d72c-e7f7-1069-1e82ac76ab41",
+  "topic": "/subscriptions/4805176c-73e4-4b8a-88cf-ea7d61b176a7/resourceGroups/rg-ga-ui-wsstatus-central/providers/Microsoft.HealthcareApis/workspaces/wsga",
```

Comment:
let's hide the details for those ids and names. for example:

/subscriptions/{subscription-id}/resourceGroups/{resource-group-name}/providers/Microsoft.HealthcareApis/workspaces/{workspace-name}